etcdctl/ctlv3/help.go

```diff
@@ -117,21 +117,17 @@ func etcdFlagUsages(flagSet *pflag.FlagSet) string {
 		} else {
 			format = "   %s   --%s"
 		}
-		if len(flag.NoOptDefVal) > 0 {
-			format = format + "["
-		}
 		if flag.Value.Type() == "string" {
 			// put quotes on the value
 			format = format + "=%q"
 		} else {
 			format = format + "=%s"
 		}
-		if len(flag.NoOptDefVal) > 0 {
-			format = format + "]"
-		}
 		format = format + "\t%s\n"
 		shorthand := flag.Shorthand
-		fmt.Fprintf(x, format, shorthand, flag.Name, flag.DefValue, flag.Usage)
+		defValue := strings.Replace(flag.DefValue, "[", "", -1)
```

Comment:
can you ask in cobra repo about why does it do this?

Reply:
I will ask, I guess something like using a plain print %v statement of the value that is expected

Reply:
see link below

Reply:
@gertcuykens hi, which link? Did you forget to paste the link :) Thanks!

Reply:
lower :) `flag.DefValue for string array #803` https://github.com/spf13/cobra/issues/803

Reply:
:) Thanks!

Reply:
Seems like they aren't actively looking at issues. I feel like not manually overriding the brackets, as usually that's a more visual mean of expressing it's a list. @xiang90  I am OK if we want to modify the text as this PR did in the ctl.go.